### DISCUSSION TITLE
Disable REST API Filter while creating the application

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5405] [FAQs]
I'm using REST API to create an application on the Ant Media Server instance. I can create the application but after creating it the REST API Security Filter is enabled by default.

Is there a way to prevent the REST API Filter from enabling by default when using API to create an application?

Thank you.

**Comment by USAMAWIZARD** [comment]
1. Create the application using REST API like you are doing currently.
2. You need to authenticate to call management Rest APIs. You can choose any of the methods defined [here](https://antmedia.io/docs/guides/developer-sdk-and-api/rest-api-guide/management-rest-apis/).
In this example, I authenticated using user name and password.

3. Get the application settings for that particular application using [this api.](https://antmedia.io/rest/#/ManagementRestService/getSettings)
4. Set the new updated application settings with IP filter disabled using [this api.](https://antmedia.io/rest/#/ManagementRestService/changeSettings)


Call the [Get Settings Rest API](https://antmedia.io/rest/#/ManagementRestService/getSettings) to get the settings of application. In this example, for newapp.
Here is the curl sample:
```
curl --location 'http://localhost:5080/rest/v2/applications/settings/newapp' \
--header 'Cookie: JSESSIONID=9762ACD603974026958AEAB9F0068663'
```
In response you will get all the application properties, that you need to copy and use it in next call.
After getting the application settings from the response of the previous Rest API call, use [Change settings Rest API](https://antmedia.io/rest/#/ManagementRestService/changeSettings) to change the settings.
For example, to disable ​ipFilterEnabled option you need to change the below property and rest other properties will be same.

` "ipFilterEnabled": false, `
Here is the curl sample:
```
curl --location 'http://localhost:5080/rest/v2/applications/settings/newapp' \
--header 'Content-Type: application/json' \
--header 'Cookie: JSESSIONID=9762ACD603974026958AEAB9F0068663' \
--data '{
    "ipFilterEnabled": false,
}'
```
